api/pkg/apps/cms/server_test.go

```diff
@@ -0,0 +1,160 @@
+// +build integration
+
+package cms_test
+
+import (
+	"context"
+	"errors"
+	. "github.com/nrc-no/core/pkg/apps/cms"
+	"github.com/nrc-no/core/pkg/generic/server"
+	"github.com/nrc-no/core/pkg/rest"
+	uuid "github.com/satori/go.uuid"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"net"
+	"net/http"
+	"os"
+	"testing"
+)
+
+type Suite struct {
+	suite.Suite
+	server     *Server
+	serverOpts *server.GenericServerOptions
+	ctx        context.Context
+	client     *ClientSet
+}
+
+func GetEnvOrDefault(key, defaultValue string) string {
+	if value, ok := os.LookupEnv(key); ok {
+		return value
+	}
+	return defaultValue
+}
+
+var (
+	ctx = context.Background()
+
+	mongoUsername = GetEnvOrDefault("MONGO_USERNAME", "root")
+	mongoPassword = GetEnvOrDefault("MONGO_PASSWORD", "example")
+	mongoHost     = GetEnvOrDefault("MONGO_HOST", "localhost:27017")
+	mongoDatabase = GetEnvOrDefault("MONGO_DATABASE", "e2e")
+)
+
+func (s *Suite) SetupSuite() {
+	// Using a random port
+	ip := net.ParseIP("127.0.0.1")
+	listener, err := net.ListenTCP("tcp", &net.TCPAddr{
+		IP: ip,
+	})
+	if !assert.NoError(s.T(), err) {
+		s.T().Fatal(err)
+		return
+	}
+	s.T().Logf("Listening at: %s", listener.Addr().String())
+	_, port, err := net.SplitHostPort(listener.Addr().String())
+	if !assert.NoError(s.T(), err) {
+		s.T().Fatal(err)
+		return
+	}
+
+	var mongoClientFn = func(ctx context.Context) (*mongo.Client, error) {
+		mongoClient, err := mongo.NewClient(options.Client().SetAuth(options.Credential{Username: mongoUsername, Password: mongoPassword}).SetHosts([]string{mongoHost}))
+		if !assert.NoError(s.T(), err) {
+			s.T().Fatal(err)
+		}
+		if err := mongoClient.Connect(ctx); err != nil {
+			logrus.WithError(err).Errorf("failed to connect to mongo")
+			return nil, err
```

Comment:
You don't need to use logrus here, if you need logging during a test, you can use 
```
s.T().Log(...
```

Reply:
Otherwise `go` doesn't know which test your logging belongs to